Dockerfile

```diff
@@ -9,9 +9,13 @@ COPY package.json .
 COPY package-lock.json .
 
 RUN npm ci
+RUN npm i -g typescript
 
 COPY . .
 
+RUN npm run build
+RUN rm -rf ./src
```

Comment:
This is fine but maybe it'd be better to copy in the next dockerfile stage what's actually necessary, which should be just the `dist` directory and the `package.json` file.

Reply:
Done